tests/testAutopas/tests/selectors/RemainderTraversalTest.h

```diff
@@ -15,12 +16,13 @@ enum ParticleStorage {
   bufferHalo,
 };
 
-class AutoTunerRemainderTraversalTest
-    : public AutoPasTestBase,
-      public ::testing::WithParamInterface<std::tuple<ParticleStorage, ParticleStorage>> {
+class RemainderTraversalTest : public AutoPasTestBase,
+                               public ::testing::WithParamInterface<std::tuple<ParticleStorage, ParticleStorage>> {
  public:
-  AutoTunerRemainderTraversalTest() = default;
-  ~AutoTunerRemainderTraversalTest() override = default;
+  RemainderTraversalTest() : numBuffers(autopas::autopas_get_num_threads()){};
```

Comment:
Yes because the number of buffers depends on the number of threads. However, `autopas_get_num_threads()` will always return 1 here. Should be `autopas_get_max_threads().

Reply:
Why does it always return 1 here? Because we run the tests only with 1 omp thread? 
Sorry, I might not fully understand this.
The point is, we should run this test for cases with more than 1 buffer.

Reply:
`autopas_get_num_threads()` returns the currently active number of threads. Since this constructor is not in an OpenMP region this will always be 1.
`autopas_get_max_threads()` returns the number of threads which is maximally available. This is the value of the `OMP_NUM_THREADS` environment variable, or if this is not set, the number of logical cores on the system (usually).

Reply:
Okay perfect, thanks!